pkgs/os-specific/linux/kmod/default.nix

```diff
@@ -14,13 +14,17 @@ in stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ autoreconfHook pkgconfig libxslt ];
-  buildInputs = [ xz /* zlib */ ];
+  buildInputs = [ xz ];
```

Comment:
This derivation requires `liblzma.so` from `xz.crossDrv.out`, so I don't think that will work.